src/routes/devices/deviceValidator.ts

```diff
@@ -29,6 +29,9 @@ export const validator = (): any => {
     check('hostname')
       .optional({ nullable: true })
       .isString(),
+    check('mpsusername')
+      .optional({ nullable: true })
```

Comment:
isn't this actually required? Otherwise we don't know the username credentials for a device to connect into MPS

Reply:
I am not sure about it. What is the device is only activated and not CIRA configured?

Reply:
done